src/gpodder/gtkui/main.py

```diff
@@ -3653,11 +3644,9 @@ def check_for_updates(self, silent):
     def on_wNotebook_switch_page(self, notebook, page, page_num):
         self.play_or_download(current_page=page_num)
         if page_num == 0:
-            # The message area in the downloads tab should be hidden
+            # The infobar in the downloads tab should be hidden
             # when the user switches away from the downloads tab
-            if self.message_area is not None:
-                self.message_area.hide()
-                self.message_area = None
+            self.resume_all_infobar.set_revealed(False)
```

Comment:
@elelay @tpikonen Should this code be removed now that the infobar can be closed? I've switched tabs many times for various reasons and then lose the resume all button. Thoughts?

Reply:
I would be fine with removing this.

Reply:
yes, when reviewing I wondered it was a regression but no: it was already there.
+1 to remove it.